docs/UserGuide.md

```diff
@@ -240,21 +304,42 @@ Format: `list`
 
 #### Finding an employee : `find`
 
-Finds employees by searching for specified keywords. You can search for a specific name, position, or department, or with any combination of the aforementioned.
+Once you get familiar with Coydir and batch-adding numerous employees into the database, using `list` simply isn't sufficient to look for the right employee.
+The `find` command is a specific, precise function meant for searching through your employee directory.
 
-The keywords for searching are case-insensitive, and need not be full words. Running the command will then display all employees whose details match the specified keywords.
+You can find employees by searching for:
 
-**Note**: There must be _at least one_ of the employees' particulars (and keywords) in the search.
+- Name
+- Position
+- Department
+
+Or any combination of the above. Coydir will then display all employees that fulfill your search criteria.
 
 Format: `find [n/NAME_KEYWORD] [j/POSITION_KEYWORD] [d/DEPARTMENT_KEYWORD]`
 
+| Field        | TAG | Description                               | Requirement  | Default Value |
+| ------------ | --- | ----------------------------------------- | ------------ | ------------- |
+| `NAME`       | n/  | Search keyword for name of employee       | **Optional** | _Blank_       |
+| `POSITION`   | j/  | Search keyword for position of employee   | **Optional** | _Blank_       |
+| `DEPARTMENT` | d/  | Search keyword for department of employee | **Optional** | _Blank_       |
+
+<div markdown="span" class="alert alert-primary">:bulb:
+  <h5>Important Tips!</h5>
+  <ul>
+    <li>There must be <em>at least one</em> parameter (and keyword) in the search, else Coydir will not know what to search for!</li>
+    <li>The keywords you input for searching are <strong>case-insensitive</strong>. Additionally, they need not be full words, just a segment will do. This means that searching for the department "tech" will show results for both "Technology and "Information Technology".</li>
+  </ul>
+</div>
+
+![Find Command - Example](./images/ui-screenshots/find-example.png)
+
 Example:
 
-- `find n/John j/engineer d/Tech` displays the employee "John Doe", who is a "Software Engineer" in the "Information Technology" department.
+- `find j/officer d/Management` displays 2 employees, who are "Officers" in the "General Management" department.
```

Comment:
Hmm for this part maybe can give more examples?

I was thinking of :
1 example of only one parameter
1 example of 2 parameters 
1 example of 3 parameters

Reply:
Yeah can add one for 3 parameters (to demonstrate usage for highly specific searches), but I think another example for 1 parameter is not necessary. 

The purpose of the 2 parameter example is firstly to show the use of the command, secondly to show the stacking of keyword matching, and lastly, to show that it is capable of returning all employees who fit the criteria. To that extent I think the example for 1 parameter is not needed here.